csl-citation.json

```diff
@@ -1,101 +1,103 @@
 {
+    "$id": "https://raw.githubusercontent.com/citation-style-language/schema/master/csl-citation.json",
+    "$schema": "http://json-schema.org/draft-07/schema#",
     "description": "JSON schema for CSL citation objects",
-    "$schema" : "http://json-schema.org/draft-03/schema#",
-    "id": "https://github.com/citation-style-language/schema/raw/master/csl-citation.json",
+    "definitions": {
+        "citationItemDefinition": {
+            "type": "object",
+            "properties": {
+                "id": {
+                    "type": [
+                        "string",
+                        "number"
+                    ]
+                },
+                "itemData": {
```

Comment:
This is not used whatsoever in citeproc-js.

Reply:
It's not?

https://github.com/rmzelle/ref-extractor/wiki#single-item-zotero-citation shows an example of how Zotero embeds CSL citation metadata in Word documents. It shows "itemData" as a field of "citationItems", and it contains an array of item metadata as defined by csl-data.json.

Reply:
Hm.. Must be a single one-off case built into citeproc-js then. Because that doesn't exist anywhere in my implementation.